docs/hub/gguf-llamacpp.md

```diff
@@ -1,16 +1,46 @@
 # GGUF usage with llama.cpp
 
-Llama.cpp directly allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
+Llama.cpp allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
```

Comment:
Should we also document where downloaded file (via `--hf-repo`) is cached? Just in case user wants to clear the cache. Ref: https://github.com/ggerganov/llama.cpp/pull/7826

Reply:
(I re-read this, `llama.cpp would download the model checkpoint in the directory you invoke it from` ==> no longer true, because we now have the cache directory, as mentioned in my last comment)

Reply:
Updated.